lib/middleware/blp-session.ts

```diff
@@ -21,7 +18,7 @@ var LOGGER: bunyan.Logger = bunyan.createLogger(conf.get('loggerOptions'));
 function createSession(sessionId: string): Promise<blpapi.Session>
 {
     if (!(sessionId in SESSION_STORE)) {
-        var s = new blpapi.Session(DEFAULT_SESSION_OPTIONS);
+        var s = new blpapi.Session(conf.get('sessionOptions'));
```

Comment:
Looking at the code from a glance, one may think that this could change for each request that comes through unless they dig deeper and look at `lib/config.ts`.  There is nothing logically wrong with this line of code; it may be confusing on what it conveys without some global knowledge of the rest of the code base.